### ISSUE TITLE
Ignore specific rules

### ISSUE THREAD

**Issue by Stephs15** [pa11y/pa11y-ci#60]
Hi! 
Is there a way to ignore a specific rule?
For example if would like to ignore WCAG2AA.Principle1.Guideline1_4.1_4_3.G18 where should I specify this?

**Comment by dominicfraser** [comment]
I've just been looking at this, and found a related issue.

To answer @Stephs15 you can do so within your `.pa11yci.json` file ([This](http://nick.freear.org.uk/2018/03/02/pa11y-ci-accessility-testing.html) is another setup guide).

```json
{
  "standard": "WCAG2AA",
  "level": "error",
  "defaults": {
    "timeout": 20000,
    "wait": 2000,
    "chromeLaunchConfig": {
      "args": ["--no-sandbox"]
    }
  },
  "urls": [
      {
        "url": "https://www.google.com",
        "viewport": { "width": 320, "height": 480 },
        "ignore": [
          "WCAG2AA.Principle2.Guideline2_4.2_4_2.H25.1.NoTitleEl",
          "WCAG2AA.Principle3.Guideline3_1.3_1_1.H57.2",
          "WCAG2AA.Principle1.Guideline1_4.1_4_3.G145",
          "WCAG2AA.Principle1.Guideline1_4.1_4_6.G18.Fail"
        ],
      }
  ]
}

```

The issue I have is: `pally-ci` does not output the specific guides that fail in the same way `pa11y` does.  This left me search through `pa11y` issues to discover that `WCAG2AA.Principle1.Guideline1_4.1_4_3.G18` is actually added as an ignored rules of `WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail` - whereas in `pa11y` main it lists the exact code that fails.

Would it be possible to output the ones that fail, as they are not always exactly the same (as proven) in the list of [CodeSniffer Rules](https://github.com/pa11y/pa11y/wiki/HTML-CodeSniffer-Rules)?

**Comment by Stephs15** [comment]
@dominicfraser Thanks for the config example. Helped me a lot with other small issued I had

**Comment by rowanmanning** [comment]
Thanks @dominicfraser for answering the question. On the other point you raised, we can investigate! Would it be possible to open a separate issue? Thanks :)
